src/book/02-system/01-foundations/04-nuclear.mdx

```diff
@@ -186,6 +186,20 @@ NBS files are NUClear Binary Stream files. To make it easier to serialise stream
 
 All values within this format are little endian.
 
+An `nbz` or `nbs.gz` file is a compressed `nbs` file using gzip.
+
+NBS files can have an acsossiated index file. This allows for quicker random access to the coresponding nbs file. An `nbs.idx` file has the folowwing frames repeated continuously, which are then compressed with gzip:
```

Comment:
```suggestion
NBS files can have an associated index file. This allows for quicker random access to the coresponding nbs file. An `.nbs.idx` file has the following frames repeated continuously, which are then compressed with gzip:
```

Also, is this saying that the .idx files are compressed already, or is compression something you have to do after you make the idx file?